src/Beberlei/Metrics/Collector/Librato.php

```diff
@@ -71,15 +97,20 @@ public function measure($variable, $value)
         );
     }
 
+    /**
+     * {@inheritDoc}
+     */
     public function flush()
     {
         if (!$this->data['gauges'] && !$this->data['counters']) {
             return;
         }
 
+        $authToken = base64_encode($this->username.':'.$this->password);
```

Comment:
Easier to debug, avoid having too many things in a single line of code that's already concatenating two strings.